apps/meteor/client/views/room/Header/ToolBox/ToolBox.tsx

```diff
@@ -32,22 +32,24 @@ const ToolBox = ({ className }: ToolBoxProps): ReactElement => {
 	const visibleActions = isMobile ? [] : actions.slice(0, 6);
 
 	const hiddenActions: Record<string, ToolboxActionConfig> = Object.fromEntries(
-		(isMobile ? actions : actions.slice(6)).map((item) => {
-			hiddenActionRenderers.current = {
-				...hiddenActionRenderers.current,
-				[item.id]: item.renderOption || renderMenuOption,
-			};
-			return [
-				item.id,
-				{
-					label: { title: t(item.title), icon: item.icon },
-					action: (): void => {
-						openTabBar(item.id);
+		(isMobile ? actions : actions.slice(6))
+			.filter((item) => !item.disabled)
```

Comment:
@ggazzo I decided to just filter here instead of showing it disabled since the options inside the Menu component seem not to support disabled options.

Any suggestion?